openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Menu, Typography } from 'antd';
+import { ItemType } from 'antd/lib/menu/hooks/useItems';
+import React from 'react';
+import { LegendProps, Surface } from 'recharts';
+
+export const getMenuItems = (items: ItemType[], defaultKey: string) => (
+  <Menu selectable defaultSelectedKeys={[defaultKey]} items={items} />
+);
+
+export const renderLegend = (props: LegendProps, total: string) => {
```

Comment:
If it's a component then it should be PascalCase and if it's a function we should rename props